obs/installation-images.spec

```diff
@@ -68,7 +68,11 @@ ExclusiveArch:  do_not_build
 %if "%{the_version}" == ""
 %error "bad version string"
 %endif
+%ifarch %arm
+%define net_repo https://download.opensuse.org/ports/%{the_arch}/distribution/leap/%{the_version}/repo/oss
```

Comment:
I would use `armv7hl` instead of `%{the_arch}` since I suspect `%{the_arch}` value would be `armv7l`

Reply:
Maybe @dirkmueller could confirm?

Reply:
hmm, it seems it used to work in the past: https://github.com/openSUSE/installation-images/commit/4788feaa282c261b21246fea914671196f8564d0 so I guess It should be ok. Sorry for the noise.

Reply:
np, in general I try to have it more generic if we add more ports, then it should need just to change condition and not url.